pandas/plotting/_core.py

```diff
@@ -1533,7 +1533,10 @@ def hexbin(self, x, y, C=None, reduce_C_function=None, gridsize=None, **kwargs):
         return self(kind="hexbin", x=x, y=y, C=C, **kwargs)
 
 
-def _get_plot_backend(backend=None):
+_backends = {}
+
+
+def _get_plot_backend(backend="matplotlib"):
```

Comment:
Not sure if I'm missing something, but are we ignoring the `plotting.backend` option now?

Also, I find the implementation a bit complex, if it's possible I'd have a function `_load_backends_from_entry_points` that is called `if not _backends` and the rest of `_get_plot_backend` should simply try the lookup and do the import.

Reply:
> Not sure if I'm missing something, but are we ignoring the plotting.backend option now?

Uhmm whoops.